extensions/typescript/src/typescriptServiceClient.ts

```diff
@@ -416,65 +496,94 @@ export default class TypeScriptServiceClient implements ITypescriptServiceClient
 		}
 
 		const shippedVersion = this.getTypeScriptVersion(this.globalTypescriptPath);
-		const localModulePath = path.join(workspace.rootPath, 'node_modules', 'typescript', 'lib', 'tsserver.js');
+		const localModulePath = this.localTypeScriptPath;
 
-		let messageShown: Thenable<MyMessageItem | undefined>;
-		if (fs.existsSync(localModulePath) && this.getTypeScriptVersion(localModulePath)) {
+		let messageShown: Thenable<MyQuickPickItem | undefined>;
+		if (localModulePath) {
 			const localVersion = this.getTypeScriptVersion(localModulePath);
 			const usingWorkspaceVersion = this.workspaceState.get<boolean>(TypeScriptServiceClient.useWorkspaceTsdkStorageKey, false);
-			messageShown = window.showInformationMessage<MyMessageItem>(
-				usingWorkspaceVersion
-					? localize(
-						'usingWorkspaceTsVersion',
-						'Using TypeScript version {0} from workspace for IntelliSense.',
-						localVersion)
-					: localize(
-						'usingVSCodeTsVersion',
-						'Using VSCode\'s TypeScript version {0} for IntelliSense.',
-						shippedVersion
-					),
+			messageShown = window.showQuickPick<MyQuickPickItem>([
 				{
-					title: localize('use', 'Use workspace version ({0})', localVersion),
+					label: localize('useWorkspaceVersionOption', 'Use Workspace Version'),
+					description: localVersion || '',
+					detail: modulePath === localModulePath ? localize('activeVersion', 'active') : '',
 					id: MessageAction.useLocal
 				}, {
-					title: localize(
-						'useVSCodeVersionOption',
-						'Use VSCode\'s version ({0})',
-						shippedVersion),
+					label: localize('useVSCodeVersionOption', 'Use VSCode\'s Version'),
+					description: shippedVersion || '',
+					detail: modulePath === this.globalTypescriptPath ? localize('activeVersion', 'active') : '',
```

Comment:
I'd omit the `active` indication. It's not adding additional information as all information is already in the place holder text.